server/courselist.rb

```diff
@@ -772,6 +773,21 @@ def check_esb(format)
     status, headers, body = call env.merge("PATH_INFO" => '/status/check')
   end
 
+  get '/status/dependencies.?:format?' do |format|
+    format = 'html' unless (format)
+    config_hash = settings.latte_config
+    app_hash = Hash.new
+    config_hash.keys.grep(/_name$/) { |name| app_hash[name] = config_hash[name] }
+    @information = app_hash
+    if format && "json".casecmp(format) == 0 then
```

Comment:
I agree the html/json might be useful for other feeds.  Let's keep an eye out to see if other likely candidates show up before extracting the code..